containers/src/Data/Graph.hs

```diff
@@ -115,7 +115,7 @@ import Data.Tree (Tree(Node), Forest)
 
 -- std interfaces
 import Data.Foldable as F
-#if MIN_VERSION_base(4,18,0)
+#if MIN_VERSION_base(4,18,0) && defined(__GLASGOW_HASKELL__)
 import qualified Data.Foldable1 as F1
```

Comment:
Foldable1 too doesn't require quantified constraints, in case you want to have it in MHS.